ui/README.md

```diff
@@ -18,18 +20,52 @@ Build for prod
 yarn build
 ```
 
-## Vuex state management
+Build/watch in dev mode
+```
+yarn watch:dev
+```
+
+Run unit tests
+```
+yarn test
+```
+
+## State management: Vuex and pinia
+*NOTE:* Vuex is being replaced by [pinia](https://pinia.vuejs.org/).
+
 This project uses [Vuex](https://next.vuex.vuejs.org/) with the modules pattern.
 Each store module has separate files for state, actions, and mutations.
 Current convention is to only call actions from components, (no mutations).
 
+Vuex stores are under `store`; pinia stores are under `stores`.
+
 ## Vue-router
 Project routes make use of [vue-router](https://next.router.vuejs.org/guide/)
 
 ## Recommended IDE Setup
 
 [VSCode](https://code.visualstudio.com/)
 
+## Developer workflow
+
+Developer workflow for development -> build -> fast deploy. There may be issues running the vite development server since it runs on a different port than OpenNMS and OpenNMS menu items and URLs will be pointing to that port instead of the vite port.
```

Comment:
```suggestion
Developer workflow for development -> build -> fast deploy. There may be issues running the vite development server since it runs on a different port than OpenNMS; OpenNMS menu items and URLs will point to that port instead of the vite port.
```